example/01_gemm/gemm_xdl_fp16.cpp

```diff
@@ -41,7 +41,6 @@ using AElementOp = ck::tensor_operation::element_wise::PassThrough;
 using BElementOp = ck::tensor_operation::element_wise::PassThrough;
 using CElementOp = ck::tensor_operation::element_wise::PassThrough;
 
-static constexpr auto GemmDefault = ck::tensor_operation::device::GemmSpecialization_t::Default;
```

Comment:
Can you copy the alias `GemmDefault` to line 47 and line 62? This was my mistake.

Reply:
This was made redundant by syncing with upstream